### DISCUSSION TITLE
Archived Workflow API `fieldSelector` syntax?

### DISCUSSION THREAD

**Discussion by kaikiat** [argoproj/argo-workflows#10934] [Q&A] [answered]
My namespace is `argo` and the name of the workflow that I want to search is `update-service`

API : `/api/v1/archived-workflows?&listOptions.fieldSelector=metadata.namespace={namespace}&listOptions.fieldSelector=metadata.name={name}&prefix={name}`

I used the api above. However, after looking at the results I am getting workflows that are not related to `update-service`. 

The results are essentially the same as listing archive workflows without specifying the `name`, i.e `/api/v1/archived-workflows?&listOptions.fieldSelector=metadata.namespace={namespace}`.

tldr; How to list a workflow based on the name ?

**Comment by kaikiat** [comment, marked as answer]
I am closing this issue.

For anyone facing this issue, instead of using the API stated above. Use this (use `,` instead)

`API : /api/v1/archived-workflows?&listOptions.fieldSelector=metadata.namespace={namespace},metadata.name={name}`
